build/vsts_build.yaml

```diff
@@ -412,111 +412,29 @@ jobs:
-    displayName: "InstallNuGetVSIX.ps1"
-    inputs:
-      scriptName: "$(System.DefaultWorkingDirectory)\\artifacts\\EndToEnd\\scripts\\InstallNuGetVSIX.ps1"
-      arguments: "-NuGetDropPath $(System.DefaultWorkingDirectory)\\artifacts\\$(VsixPublishDir) -FuncTestRoot $(System.DefaultWorkingDirectory)\\artifacts -NuGetVSIXID $(NuGetVsixId) -ProcessExitTimeoutInSeconds 180 -VSVersion 16.0"
-      failOnStandardError: "false"
-
-  - task: PowerShell@1
-    displayName: "Collect VS Logs"
-    inputs:
-      scriptType: "inlineScript"
-      inlineScript: |
-        Invoke-WebRequest -Uri $(CollectExeUrl) -OutFile $(System.DefaultWorkingDirectory)\\VSCollect.exe
-        $(System.DefaultWorkingDirectory)\\VSCollect.exe -zip:$(System.DefaultWorkingDirectory)\\artifacts\\EndToEnd\\e2e-collectlogs.zip
-    condition: "failed()"
-
-  - task: PowerShell@1
-    displayName: "RunFunctionalTests.ps1 (continue on error)"
-    timeoutInMinutes: 75
-    continueOnError: "true"
-    inputs:
-      scriptName: "$(System.DefaultWorkingDirectory)\\artifacts\\EndToEnd\\scripts\\RunFunctionalTests.ps1"
-      arguments: "-PMCCommand $(EndToEndTestCommandToRun) -PMCLaunchWaitTimeInSecs 30 -EachTestTimoutInSecs 600 -NuGetDropPath $(System.DefaultWorkingDirectory)\\artifacts -FuncTestRoot $(System.DefaultWorkingDirectory)\\artifacts -RunCounter $(Build.BuildNumber) -VSVersion 16.0"
-    condition: "eq(variables['IsOfficialBuild'], 'true')"
-
-  - task: PowerShell@1
-    displayName: "RunFunctionalTests.ps1 (stop on error)"
-    timeoutInMinutes: 75
-    continueOnError: "false"
-    inputs:
-      scriptName: "$(System.DefaultWorkingDirectory)\\artifacts\\EndToEnd\\scripts\\RunFunctionalTests.ps1"
-      arguments: "-PMCCommand $(EndToEndTestCommandToRun) -PMCLaunchWaitTimeInSecs 30 -EachTestTimoutInSecs 600 -NuGetDropPath $(System.DefaultWorkingDirectory)\\artifacts -FuncTestRoot $(System.DefaultWorkingDirectory)\\artifacts -RunCounter $(Build.BuildNumber) -VSVersion 16.0"
-    condition: "not(eq(variables['IsOfficialBuild'], 'true'))"
-
-  - task: PublishTestResults@2
-    displayName: "Publish Test Results"
-    inputs:
-      testRunner: "JUnit"
-      testResultsFiles: "*.xml"
-      searchFolder: "$(System.DefaultWorkingDirectory)\\testresults"
-      mergeTestResults: "true"
-      testRunTitle: "NuGet.Client EndToEnd Tests On Windows"
-    condition: "succeededOrFailed()"
-
-  - task: PowerShell@1
-    displayName: "Initialize Git Commit Status on GitHub"
-    inputs:
-      scriptType: "inlineScript"
-      arguments: "-VstsPersonalAccessToken $(VstsPersonalAccessToken)"
-      inlineScript: |
-        . $(Build.Repository.LocalPath)\\scripts\\utils\\PostGitCommitStatus.ps1
-        SetCommitStatusForTestResult -PersonalAccessToken $(NuGetLurkerPersonalAccessToken) -VstsPersonalAccessToken $(VstsPersonalAccessToken) -CommitSha $(Build.SourceVersion) -TestName "EndToEnd Tests On Windows"
-    condition: "always()"
+- job: E2E_Tests_On_Windows_Part2
```

Comment:
I wouldn't be surprised if it's yet another limitation in AzDO's yaml, but is it possible to make `Part` a matrix build, so we don't have so much duplication in all the variables, pool, demands, template name

Reply:
Actually, it's not the limitation for the length of job name, but I couldn't see the full job name when the build is running..
I reconsidered it and I don't think it's important enough to adjust the length of job name. (as there is no complain about the long name `Functional_Tests_On_Windows IsCore`)
So I changed it back.